### PR TITLE
[New][Author] Add equalizecss@1.2.1 w/ npm auto-update

### PR DESCRIPTION
**Pull request by VladimirHumeniuk**
**Pull** request for issue: #10954 

# Profile of the lib
 * Git repository (required): https://github.com/VladimirHumeniuk/equalizecss
 * Official website (optional, not the repository): https://equalizecss.com/
 * NPM package url (optional): http://npmjs.com/equalizecss
 * License and its reference: https://creativecommons.org/licenses/by-sa/4.0/
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 8
   - Count of stars: 52
   - Count of forks: 6
 * NPM download stats (optional):
   - Downloads in the last month: 217

# Essential checklist
 * [*] I'm the author of this library
   * [*] I would like to add link to the page of this library on CDNJS on website / readme
 * [*] This lib was not found on cdnjs repo
 * [*] No already exist / duplicated issue and PR

# Git commit checklist
 * [*] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [*] The parent of the commit(s) in the PR is not old than 3 days.

@PeterDaveHello

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@VladimirHumeniuk the `package.json` of the library is missing, can you please add it in its folder?

**Comment by sufuf3** [comment]
@VladimirHumeniuk Thank you for your contribution.
Would you add the `package.json` as the followings?
```js
{
  "name": "equalizecss",
  "version": "1.2.1",
  "description": "css-grid built on the properties of flexboxes and written on the sass",
  "filename": "css/equalize.min.css",
  "repository": {
    "type": "git",
    "url": "git://github.com/vladimirhumeniuk/equalizecss.git"
  },
  "author": "Vladimir Humeniuk",
  "license": "MIT",
  "keywords": [
    "html",
    "framework",
    "flexbox",
    "grid",
    "css"
  ],
  "homepage": "https://equalizecss.com/",
  "npmName": "equalizecss",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "css/**/*"
      ]
    }
  ]
}
```
But here are some questions need to discuss with you.
1. Which license is this lib used? [ISC](https://github.com/VladimirHumeniuk/equalizecss/blob/master/package.json#L15) or [MIT](https://github.com/VladimirHumeniuk/equalizecss/blob/master/LICENSE)
2. Would you let `equalize.css` and `equalize.min.css` under `ajax/libs/equalizecss/1.2.1/css`? Because of the CDNJS's rule.

Thank you.

**Comment by VladimirHumeniuk** [comment]
@PeterDaveHello done

**Comment by VladimirHumeniuk** [comment]
@sufuf3 already did. MIT licence, already fix this mistake. Thanks.

**Comment by PeterDaveHello** [comment]
Let me rebase and squash those commits first, and let's see if you can pass the CI.

File: ajax/libs/equalizecss/1.2.1/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (PeterDaveHello) [inline ajax/libs/equalizecss/1.2.1/package.json]: This file should be outside the version directory.

File: ajax/libs/equalizecss/1.2.1/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "grid",
+    "css"
+  ],
+  "homepage": "https://equalizecss.com/",
+  "npmName": "equalizecss",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "css/**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/equalizecss/1.2.1/package.json]: I think we should just exclude `sass` here? cc @sufuf3 @extend1994

**Comment by VladimirHumeniuk** [comment]
@PeterDaveHello done

**Comment by PeterDaveHello** [comment]
@VladimirHumeniuk Nice, let me squash and rebase again, and @sufuf3 @extend1994 will review in detail!

**Comment by sufuf3** [comment]
Yes I get them from npm, the file structure as the followings:
```bash
equalizecss/
├── bower.json
├── package.json
├── README.md
├── src
├── src\css
├── src\css\equalize.css
├── src\css\equalize.min.css
├── src\sass
├── src\sass\equalize.sass
└── src\sass\_vars.sass
```

**Comment by sufuf3** [comment]
@extend1994 I update v1.0.82 please help review this PR thanks.

**Comment by VladimirHumeniuk** [comment]
@extend1994 is everything okay with file structure now?

**Pull request merged**